src/__experimental__/components/radio-button/radio-button.stories.js

```diff
@@ -16,7 +16,7 @@ function makeStory(name, themeSelector) {
     return (
       <RadioButtonGroup
         groupName='frequency'
-        label={ text('groupLabel', 'Please select a frequency from the options below') }
+        legend={ text('groupLabel', 'Please select a frequency from the options below') }
```

Comment:
```suggestion
        legend={ text('legend', 'Please select a frequency from the options below') }
```

Reply:
resolved in [https://github.com/Sage/carbon/pull/2395/commits/8ad77ae94edd822af3a2c8bee9159daa445c16b8](8ad77ae)